src/test/java/com/celements/model/classes/fields/EntityReferenceFieldTest.java

```diff
@@ -27,8 +28,7 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.